src/components/Class/ClassCard/index.js

```diff
@@ -12,12 +12,14 @@ import Modal from "../../common/Modal";
 
 toast.configure();
 
-function ClassCard({ item, editClass, enroll, style, indicator }) {
+function ClassCard({ item, editClass, enroll, style }) {
   const dispatch = useDispatch();
   const [enrollShowModel, setEnrollShowModel] = React.useState(false);
   const [unenrollShowModel, setunenrollShowModel] = React.useState(false);
   const [showModel, setShowModel] = React.useState(false);
```

Comment:
@Poonam-Singh-Bagh Rename it as Model as Modal everywhere

Reply:
Sure! Will rename it.